### PR TITLE
Adds a new Fungal Growth event, makes fungus re-harvestable

### PR DESCRIPTION
**Pull request by lewcc**
## What Does This PR Do
Adds a new mundane event, Fungus Growth, which causes fungus tiles to grow on maintenance tiles. 

Fungus is also now re-harvestable after some time (between 5-10 minutes).

## Why It's Good For The Game
Spacecillin is now more useful than ever with germs and infections becoming more prevalent (hello, burn wounds), but most chemists won't be producing it since fungus is pretty uncommon. This adds a little bit of life to maintenance (maybe there's some new fungus popping up), while also making it a bit more consistent to get -- assuming, of course, the event rolls. It also allows you to farm the fungus that does currently exist, though you'll have to know the spots where it spawns on maps to do so effectively.
Of course, if it pops up in unsightly spots, it can be cleaned off with soap or other common cleaning tools.

## Testing
Triggered the event, spawned some fungus among us. Scooped it off, examined it.

## Changelog
:cl:
add: A new event, "Fungal Growth", which causes space fungus to randomly pop up in maintenance.
tweak: Space fungus now regrows some time after being scooped up.
/:cl:

### CODE REVIEW

File: code/game/objects/effects/decals/Cleanable/misc_cleanables.dm [hunk 1]
```diff
@@ -277,6 +277,36 @@
 	icon_state = "flour"
 	color = "#D5820B"
 	scoop_reagents = list("fungus" = 10)
+	no_clear = TRUE
+	var/timer_id
+
+/obj/effect/decal/cleanable/fungus/examine(mob/user)
+	. = ..()
+	if(no_scoop)
+		. += "<span class='notice'>There's really not a lot here, you probably couldn't harvest anything if you tried.</span>"
```
Comment (JimKil3) [inline code/game/objects/effects/decals/Cleanable/misc_cleanables.dm:286]:
```suggestion
		. += "<span class='notice'>There's not a lot here, you probably wouldn't be able to harvest anything useful.</span>"
```
Examine text should be concise IMO but this is pretty nitpicky

File: code/game/objects/effects/decals/Cleanable/misc_cleanables.dm [hunk 2]
```diff
@@ -277,6 +277,36 @@
...
+	else
+		. += "<span class='notice'>There's a good bit here, it looks like you could scrape some off into a beaker.</span>"
```
Comment (JimKil3) [inline code/game/objects/effects/decals/Cleanable/misc_cleanables.dm:288]:
```suggestion
		. += "<span class='notice'>There's enough here to scrape into a beaker.</span>"
```
As above

File: code/game/objects/effects/decals/Cleanable/misc_cleanables.dm [hunk 3]
```diff
@@ -277,6 +277,36 @@
...
+
+/obj/effect/decal/cleanable/fungus/on_scoop()
+
+	// can you feel your heart burning?
+	// can you feel the struggle within?
+	// the fear within me is beyond anything your soul can make.
+	// you can't kill me in a way that matters
+
```
Comment (JimKil3) [inline code/game/objects/effects/decals/Cleanable/misc_cleanables.dm:296]:
```suggestion
```
funny but unnecessary

**Review by JimKil3 — CHANGES_REQUESTED** [review]
Mostly nitpicky stuff from me. After this we can remove spaceacillin from nanomeds 👍

File: code/game/objects/effects/decals/Cleanable/misc_cleanables.dm (hunk 1 above)
Comment (lewcc) [reply code/game/objects/effects/decals/Cleanable/misc_cleanables.dm:286]: I was having trouble wording that, appreciate it

**Review by S34NW — APPROVED** [review]

**Pull request merged**